src/components/ContentHighlights/data/constants.js

```diff
@@ -1,5 +1,17 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable import/no-extraneous-dependencies */
 import { faker } from '@faker-js/faker';
+import { configuration } from '../../../config';
+/* eslint-enable import/no-extraneous-dependencies */
+
+// Generate URLs for about pages from the enterprise learner portal
+export const generateAboutPageUrl = (enterpriseSlug, contentType, contentKey) => {
```

Comment:
nit: might recommend changing the function signature here to use an object instead, so that when you pass the args, it's more obvious which argument is for what, e.g.:

```jsx
const generateAboutPageUrl = ({ enterpriseSlug, contentType, contentKey }) => { ... };
```

```jsx
generateAboutPageUrl({
  enterpriseSlug: 'sluggy',
  contentType: 'course',
  contentKey: 'edX+DemoX',
});
```

Reply:
Similar to a comment below; this is more a util function than a constant; should it be moved to a `utils.js` file?